test/unit/module/complex/operators.cpp

```diff
@@ -6,25 +6,9 @@
 **/
 //==================================================================================================
 #include "test.hpp"
+#include "measures.hpp"
```

Comment:
Do we by any chance want to test against std complex with like a bunch of random inputs?